node/node.go

```diff
@@ -76,8 +76,12 @@ func NewNode(ctx context.Context, conf config.NodeConfig, nodeKey crypto.PrivKey
 		return nil, err
 	}
 
-	// TODO(tzdybal): change after implementing https://github.com/celestiaorg/optimint/issues/67
-	baseKV := store.NewInMemoryKVStore()
+	var baseKV store.KVStore
+	if conf.RootDir == "" && conf.DBPath == "" { // this is used for testing
+		baseKV = store.NewInMemoryKVStore()
```

Comment:
Maybe at least log a warning here?

Reply:
I clicked green button too fast... Added warning in #82.